public/video-ui/src/pages/Video/tabs/Pluto.jsx

```diff
@@ -29,35 +25,46 @@ export class PlutoTabPanel extends React.Component {
     return (
       <TabPanel {...rest}>
         <div className="form__group">
-          {
-            video.plutoData &&
-              <PlutoProjectLink projectId={video.plutoData.projectId}/>
-          }
+          {video.plutoData && (
+            <PlutoProjectLink projectId={video.plutoData.projectId} />
+          )}
 
           <header className="video__detailbox__header">Commission</header>
-          <ReadOnlyPlutoItem id={video.plutoData && video.plutoData.commissionId} itemType="commission" />
+          <ReadOnlyPlutoItem
+            id={video.plutoData?.commissionId}
+            itemType="commissions"
+          />
 
           <header className="video__detailbox__header">Project</header>
-          <ReadOnlyPlutoItem id={video.plutoData && video.plutoData.projectId} itemType="project" />
-
+          <ReadOnlyPlutoItem
+            id={video.plutoData?.projectId}
+            itemType="projects"
+          />
         </div>
       </TabPanel>
     );
   }
 }
 
-const ReadOnlyPlutoItem = ({id, itemType}) => {
+/**
+ *
+ * @param {{id: string, itemType: import('../../../services/PlutoApi').PlutoItemType}} param0
+ * @returns
+ */
+const ReadOnlyPlutoItem = ({ id, itemType }) => {
+  const [title, setTitle] = useState(id ? 'Loading...' : '');
 
-  const [ title, setTitle ] = useState(id ? "Loading..." : "");
-
-  if(id) {
```

Comment:
React forbids us from running hooks conditionally, so I've moved the condition to inside the hook callback